assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -39,7 +39,8 @@ import { ActionLabel } from './types';
 const PreviewPane = styled.div`
   position: relative;
   border-radius: ${({ theme }) => theme.storyPreview.borderRadius}px;
-  height: ${({ cardSize }) => `${cardSize.height}px`};
+  height: ${({ cardSize }) =>
+    `${cardSize.height + cardSize.dangerZoneHeight}px`};
```

Comment:
So we want the preview part of the grid item to be the new 9 : 16 ratio PLUS the danger zone, which is separate from the two thirds, clean and safe story - it is just overflow that needs to be allowed on the screen. Since we are using a css grid here we want to force this height into action (like we had been). If we update the height itself with the dangerzone instead of doing it this way then the unitsProvider things the card should be height + dangerZoneHeight and the problem of the missing dangerzone area continues while our cards get really tall.

Reply:
Thank you for going through this. I totally agree and that's why i switched the PR back to draft after Dima explained what was happening. I'm sorry if I didn't communicate that well. Now the wrappers are set to 9:16 and the story itself is maintaining 2:3 _with_ overflow so that the fullbleed at 9:16 comes through. This is what the editor side of things does as well.